policyengine_us/variables/gov/states/nh/tax/income/nh_taxable_income/nh_taxable_income.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class nh_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Hampshire taxable income"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NH
+
+    def formula(tax_unit, period, parameters):
```

Comment:
Doesn't need a formula
```suggestion
    adds = ["dividend_income", "interest_income"]
```